internal/db/connection.go

```diff
@@ -3,29 +3,43 @@ package db
 import (
 	"fmt"
 	"github.com/RHEnVision/provisioning-backend/internal/config"
-	pgx "github.com/jackc/pgx/v4"
+	"github.com/jackc/pgx/v4"
 	"github.com/jackc/pgx/v4/log/zerologadapter"
+	_ "github.com/jackc/pgx/v4/stdlib"
 	"github.com/jmoiron/sqlx"
 	"github.com/pkg/errors"
 	"github.com/rs/zerolog/log"
+	"net/url"
 )
 
 var (
 	// DB is the main connection pool (sqlx on top of database/sql connection pool)
 	DB *sqlx.DB
 )
 
+func GetConnectionString(prefix string) string {
+	if len(config.GetLoggingConfig().Database.Password) > 0 {
+		return fmt.Sprintf("%s://%s:%s@%s:%d/%s",
+			prefix,
+			url.QueryEscape(config.GetLoggingConfig().Database.User),
+			url.QueryEscape(config.GetLoggingConfig().Database.Password),
+			config.GetLoggingConfig().Database.Host,
+			config.GetLoggingConfig().Database.Port,
+			config.GetLoggingConfig().Database.Name)
+	} else {
+		return fmt.Sprintf("%s://%s@%s:%d/%s",
+			prefix,
+			url.QueryEscape(config.GetLoggingConfig().Database.User),
+			config.GetLoggingConfig().Database.Host,
+			config.GetLoggingConfig().Database.Port,
+			config.GetLoggingConfig().Database.Name)
+	}
+
+}
```

Comment:
The prefix is needed because golang-migrations has its own format of database URL (pgx vs postgres). But we still use pgx in both cases.